projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.service.ts

```diff
@@ -41,7 +41,8 @@ export class WaterfallChartService {
         $$spanName: {
           name: datum.name,
           serviceName: datum.serviceName,
-          protocolName: datum.protocolName
+          protocolName: datum.protocolName,
+          error: datum.errorCount > 0
```

Comment:
nit: `hasError` would convey it is boolean and not an error string

Reply:
Yeah, 
I have made the changes